tests1/myfile1.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+echo $_POST[ 'phpcs should catch this problem' ];
```

Comment:
:no_entry_sign: **Error**: Detected usage of a possibly undefined superglobal array index: $_POST['phpcs should catch this problem']. Use isset() or empty() to check the index exists before using it (*WordPress.Security.ValidatedSanitizedInput.InputNotValidated*).